index.html

```diff
@@ -1,63 +1,147 @@
-                <li class="common-li">Bootstrap</li>
-                <li class="common-li">Ruby on Rails</li>
-                <li class="common-li">RSPec</li>
-                <li class="common-li">CapyBara</li>
-                <li class="common-li">Selenium</li>
-            </ul>
-            <ul>
-                <li class="blue1">Skills</li>
-                <li class="common-li">Codekit</li>
-                <li class="common-li">Github</li>
-                <li class="common-li">Codepen</li>
-                <li class="common-li">Gitlab</li>
-                <li class="common-li">Terminal</li>
-            </ul>
-            </div>
-            <hr class="line">
-            <ul class="icons">
-                <li><a href="#"><i class="fa fa-github bars"></i></a></li>
-                <li><a href="#"><i class="fa fa-twitter bars"></i></a></li>
-                <li><a href="#"><i class="fa fa-linkedin bars"></i></a></li>
-                <li><a href="#"><i class="fa fa-angellist bars"></i></a></li>
-                <li><a href="#"><i class="fa fa-medium bars"></i></a></li>
-            </ul>
-        </div>      
-    </body>
+
+<body>
+  <header class="nav">
+    <a class="logo" href="#">Nicholas</a>
+    <nav>
+     <i class="fas fa-bars bars"></i>
+ </nav>
+</header>
+<section class="headers">
+     <h1>Nicholas Mutale</h1> 
+ <h2>Software Developer</h2>
+ <p>I can help you build a product , feature or 
+     website Look through some of my work and experience! 
+     If you like what you see and have a project you need coded, 
+     don’t hestiate to contact me.</p>
+     <hr class="line">
+     <div class="software">
+     <ul>
+         <li class="blue1">Language</li>
+         <li class="common-li">Javascript</li>
+         <li class="common-li">Ruby</li>
+         <li class="common-li">Html</li>
+         <li class="common-li">Css</li>
+     </ul>
+     <ul>
+         <li class="blue1">Frameworks</li>
+         <li class="common-li">Bootstrap</li>
+         <li class="common-li">Ruby on Rails</li>
+         <li class="common-li">RSPec</li>
+         <li class="common-li">CapyBara</li>
+         <li class="common-li">Selenium</li>
+     </ul>
+     <ul>
+         <li class="blue1">Skills</li>
+         <li class="common-li">Codekit</li>
+         <li class="common-li">Github</li>
+         <li class="common-li">Codepen</li>
+         <li class="common-li">Gitlab</li>
+         <li class="common-li">Terminal</li>
+     </ul>
+     </div>
+     <hr class="line">
+     <ul class="icons">
+         <li><a href="#"><i class="fa fa-github bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-twitter bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-linkedin bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-angellist bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-medium bars"></i></a></li>
+     </ul>
+</section>      
+
+  <section class="header">
+    <h2 class="recent">MY RECENT WORK</h2>
```

Comment:
- [ ] Before this text, you miss worktext with an arrow. Please add it.
### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/148649198-b6bd17ea-9d49-40f3-a456-6047966ba2da.png)

Reply:
Addressed 🔥